### DISCUSSION TITLE
Full calendar "theme" using shadcn-svelte components?

### DISCUSSION THREAD

**Discussion by victorDigital** [huntabyte/shadcn-svelte#656] [Help]
Does anyone know a type of full calendar that supports the shadcn components.

Currently my project is using a DaisyUI-styled calendar from [This full calendar svelte port](https://github.com/vkurko/calendar) 
![image](https://github.com/huntabyte/shadcn-svelte/assets/99267380/c9c22773-124b-4548-9308-75c9bbc5317b)

I'm working on rebuilding the page in shadcn-svelte any suggestions to libs or solutions are greatly appreciated

❤️ from 🇩🇰

**Comment by victorDigital** [comment]
welp

**Reply by Extarys** [reply]
Have you found something on this? :) I'm also looking into it.

**Reply by victorDigital** [reply]
Not really, I just resorted to using the raw tailwind classes for the card and button in my own components

**Reply by Temuujin-Munkhtsetseg** [reply]
Hi. Looks like the thread has died but I'm guessing no one has found one yet? :')
